src/LoginForm/LoginForm.css

```diff
@@ -1,4 +1,28 @@
 
+.login-screen {
+  width: 100vw;
+  height: 100vh;
+  display: flex;
+  flex-flow: column nowrap;
+  background: url(../assets/background.jpg) no-repeat;
```

Comment:
I like the moving of the background to the login form instead of the main- makes more sense to me this way.